stakingModel/test/UBIScheme.e2e.test.js

```diff
@@ -0,0 +1,192 @@
+  const transaction = await registrar.proposeScheme(avatarAddress, addr, NULL_HASH, "0x00000010", NULL_HASH);
+  proposalId = transaction.logs[0].args._proposalId;
+  const voteResult = await absoluteVote.vote(proposalId, 1, 0, fnd);
+  return voteResult.logs.some(e => e.event === 'ExecuteProposal');
+}
+
+async function increaseDays(days=1) {
+  const id = await Date.now();
+  const duration = days*86400;
+  await web3.currentProvider.send(
+    {
+      jsonrpc: "2.0",
+      method: "evm_increaseTime",
+      params: [duration],
+      id: id + 1
+    },
+    () => {}
+  );
+  await web3.currentProvider.send(
+    {
+      jsonrpc: "2.0",
+      method: "evm_mine",
+      id: id + 1
+    },
+    () => {}
+  );
+};
+
+contract("UBIScheme - network e2e tests", ([founder, claimer, fisherman]) => {
+  let dai, cDAI, simpleStaking, goodReserve, goodFundManager, goodDollar, marketMaker, contribution, controller, ubi, firstClaimPool, identity;
+  let deploy_settings, ubiBridgeRecipient, avatarAddress, registrar, absoluteVote, proposalId, setReserve, addMinter;
+
+  before(async function() {
+    const staking_file = await fse.readFile("releases/deployment.json", "utf8");
+    const dao_file = await fse.readFile("../releases/deployment.json", "utf8");
+    const staking_deployment = await JSON.parse(staking_file);
+    const dao_deployment = await JSON.parse(dao_file);
+    const staking_addresses = staking_deployment[NETWORK];
+    const dao_addresses = dao_deployment[NETWORK];
+    avatarAddress = dao_addresses.Avatar;
+    ubiBridgeRecipient = staking_addresses.UBIScheme;
+    dai = await DAIMock.at(staking_addresses.DAI);
+    cDAI = await cDAIMock.at(staking_addresses.cDAI);
+    simpleStaking = await SimpleDAIStaking.at(staking_addresses.DAIStaking);
+    goodReserve = await GoodReserve.at(staking_addresses.Reserve);
+    goodFundManager = await GoodFundsManager.at(staking_addresses.FundManager);
+    marketMaker = await MarketMaker.at(staking_addresses.MarketMaker);
+    contribution = await ContributionCalculation.at(staking_addresses.Contribution);
+    controller = await Controller.at(dao_addresses.Controller);
+    ubi = await UBI.at(staking_addresses.UBIScheme);
+    firstClaimPool = await FirstClaimPool.at(staking_addresses.FirstClaimPool);
+    identity = await Identity.at(dao_addresses.Identity);
+    goodDollar = await GoodDollar.at(dao_addresses.GoodDollar);
+    registrar = await SchemeRegistrar.at(dao_addresses.SchemeRegistrar);
+    absoluteVote = await AbsoluteVote.at(dao_addresses.AbsoluteVote);
+    deploy_settings = await fse.readFile("../migrations/deploy-settings.json", "utf8");
+    // schemes
+    setReserve = await FundManagerSetReserve.new(avatarAddress, goodFundManager.address, goodReserve.address);
+    addMinter = await AddMinter.new(avatarAddress, goodReserve.address);
+  });
+  
+  it("should start the pool", async () => {
+    let isActive = await firstClaimPool.isActive();
+    if (!isActive)
```

Comment:
this is e2e you test the state you expect. either it should be inactive on deploy or active

Reply:
db43a7633912fed7aa89a7ff580fbe5238d650b5